src/lib/authselect_check.c

```diff
@@ -26,6 +26,43 @@
 #include "lib/authselect_files.h"
 #include "lib/authselect_private.h"
 
+static bool
+check_content(const char *content, const char *expected)
+{
+    const char *nextline;
+
+    /* Each generated file contains a "generated by" preamble that consists
+     * of two comment lines and one empty line. */
```

Comment:
Not really, since the preamble contains also a data which is generated from time(NULL). The proper way would be to check the text and also parse the date format, but since it is just a comment with no effect, I believe this is enough. I explained it in the code comment.